src/pano.tsx

```diff
@@ -88,7 +88,7 @@ class Pano extends Component<PanoProps, PanoState> {
 
 	loadTexture() {
 		this.texture = this.loader.load(
-			process.env.PUBLIC_URL + "resource/" + this.currLoc.fname,
+			require(`./assets/viewPano/resource/${this.currLoc.fname}`),
```

Comment:
You can use this as an example on how you can import local img files (including svg). Importing svg should work as I remember CRA configured webpack to handle it. If not, let me know.